### PR TITLE
replacing WebOps with Site Reliability Engineer ad

### PR DESCRIPTION
**Pull request by moyni**
@nlopes @Sinjo Happy with the content?

### CODE REVIEW

File: app/pages/about/jobs/positions/site-reliability-engineer.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          tens of thousands of transactions every day. This means we need to make
+          sure our systems and infrastructure scale adequately not only to keep
+          up with demand, but to anticipate it. Businesses across the UK and Europe
+          trust us with their payments, so it’s crucial that we can provide
+          consistent performance and excellent reliability.
+        </p>
+        <p className='para'>
+          We are looking for engineers to design new systems and continually improve
+          our existing systems to ensure we are resilient to failure, and can tackle
+          any complex cloud infrastructure problems as they emerge.
```
Comment (nlopes) [inline app/pages/about/jobs/positions/site-reliability-engineer.js]: Probably remove cloud.

File: app/pages/about/jobs/positions/site-reliability-engineer.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        </p>
+        <h4 className='u-text-s u-margin-Bxs u-margin-Tm'>The role</h4>
+        <p className='para'>
+          As one of our Site Reliability Engineers at GoCardless you will be part
+          of a small team that sets the direction of our platform and codebase.
+          You’ll have ownership over your work and you will make decisions around
+          new infrastructure and which technologies are the best suited to help
```
Comment (nlopes) [inline app/pages/about/jobs/positions/site-reliability-engineer.js]: "are ~~the~~ best suited" maybe?

**Comment by moyni** [comment]
@Sinjo could you take a look at this version, including changes from @nlopes

File: app/pages/about/jobs/positions/site-reliability-engineer.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        <p className='para'>Some projects we have recently worked on:</p>
+        <ul className='list'>
+          <li><a href='https://gocardless.com/blog/zero-downtime-postgres-migrations-the-hard-parts/'>Dealing
+          with the realities of running a relational database at scale</a></li>
+          <li>Building a high-availability PostgreSQL cluster</li>
+          <li><a href='https://gocardless.com/blog/in-search-of-performance-how-we-shaved-200ms-off-every-post-request/'>improving
+          performance with investigative work</a></li>
+        </ul>
+        <p className='para'>
+          Over the coming months, as we continue launching our
```
Comment (nlopes) [inline app/pages/about/jobs/positions/site-reliability-engineer.js]: Over the coming months we will be launching our service across new geographies. You'll help ensure we can operate effectively across the world and how we can continue to provide [...]

File: app/pages/about/jobs/positions/site-reliability-engineer.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        <p className='para'>
+          We want to work with people who are always trying to improve and automate
+          processes, who enjoy finding elegant solutions for complex problems,
+          who thrive with new challenges and cryptic issues, who believe there
+          is always something new to learn and that improvement is an iterative
+          process.
+        </p>
+        <p className='para'>You should apply if you have:</p>
+        <ul className='list'>
+          <li>experience with systems internals, and knowledge of Unix fundamentals</li>
```
Comment (nlopes) [inline app/pages/about/jobs/positions/site-reliability-engineer.js]: We have all the other lists entries capitalized, this one should be as well

**Comment by nlopes** [comment]
:+1:

File: app/pages/about/jobs/positions/site-reliability-engineer.js [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        <p className='para'>Not essential, but nice to have:</p>
+        <ul className='list'>
+          <li>Formal education in Computer Science</li>
+          <li>Strong operational knowledge of PostgreSQL or similar RDBMS</li>
+          <li>Fluency in one or more of C, Python, Go, Ruby</li>
+        </ul>
+        <p className='para'>This role would ideally suit someone with previous
+          experience in a similar role, however we welcome a wide diversity of
+          candidates from varying backgrounds. If you’re unsure, please apply.
+          We offer competitive salary and options package commensurate with your
```
Comment (Sinjo) [inline app/pages/about/jobs/positions/site-reliability-engineer.js]: "We offer a"

File: app/pages/about/jobs/positions/site-reliability-engineer.js (hunk 4 above)
Comment (moyni) [inline app/pages/about/jobs/positions/site-reliability-engineer.js]: I disagree: the items here complete the sentence so its better grammar to begin with lowercase. If you'd prefer a capitalised list, I'll rephrase.

File: app/pages/about/jobs/positions/site-reliability-engineer.js [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+          day.
+        </p>
+        <p className='para'>What you’ll do here:</p>
+        <ul className='list'>
+          <li>Design software to improve availability, scalability and efficiency.</li>
+          <li>Solve problems and automate responses for recurrent issues.</li>
+          <li>Improve performance through troubleshooting and investigative work.</li>
+          <li>Design and implement deployment management for our systems.</li>
+          <li>Manage and monitor our platform as it grows.</li>
+          <li>Tackle every issue that comes your way when you’re on call.</li>
```
Comment (Sinjo) [inline app/pages/about/jobs/positions/site-reliability-engineer.js]: I might be reading this in an overly sensitive way, but "Tackle every issues" comes across a bit "You're on your own" to me.

How do you feel about:

"Tackle issues that come your way when you're on call."?

File: app/pages/about/jobs/positions/site-reliability-engineer.js [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+        <p className='para'>
+          We want to work with people who are always trying to improve and automate
+          processes, who enjoy finding elegant solutions for complex problems,
+          who thrive with new challenges and cryptic issues, who believe there
+          is always something new to learn and that improvement is an iterative
+          process.
+        </p>
+        <p className='para'>You should apply if you have:</p>
+        <ul className='list'>
+          <li>experience with systems internals, and knowledge of Unix fundamentals</li>
```
Comment (Sinjo) [inline app/pages/about/jobs/positions/site-reliability-engineer.js]: We should either end list items with a '.' on all lists, or drop them.

**Comment by moyni** [comment]
All changes in, inc lists.

**Comment by Sinjo** [comment]
:tada:

**Pull request merged**